### ISSUE TITLE
Add an option to xbps-pkgdb to automatically re-install packages with errors

### ISSUE THREAD

**Issue by ericonr** [void-linux/xbps#295]
`xbps-pkgdb -a` only lists packages with issues, requiring some smart shell scripting to get a list of all affected packages to try and reinstall. It should have a flag to do this automatically, like:

- `--fix -f`: shows a xbps-install dialog after running the checks, which you can then accept
- `--yes -y`: when used with `-f`, reinstalls affected packages automatically, without any user input

Ideally, have an option that will only list the affected packages, without specifying each file.

**Comment by Duncaen** [comment]
Reinstall is only useful for files and metadata check, for all other issues might just not work.

**Comment by ericonr** [comment]
Do you think it would still make sense to have reinstalling as an option for those cases?

**Comment by Duncaen** [comment]
I don't really think its worth it, maybe a different output format that just lists package names and maybe a way to specify which checks should be run seem like a nice addition which would make it easier to one line your way around with xbps-pkgdb and xbps-install.

**Comment by ericonr** [comment]
Fair enough. So something like: `--checks=file,metadata` and `--list -l`?

**Comment by ericonr** [comment]
Or `--pkg-names` instead of list.

**Comment by camoz** [comment]
Or `--list-pkgs`.
